frontend/app/components/comment-form/text-expander.module.pcss

```diff
@@ -0,0 +1,61 @@
+.suggester {
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 30;
+  min-width: 180px;
+  padding: 0;
+  margin: 24px 0 0;
+  list-style: none;
+  cursor: pointer;
+  background: #fff;
+  border: 1px solid #dfe2e5;
+  border-radius: 3px;
+  box-shadow: 0 1px 5px rgba(27, 31, 35, 0.15);
+}
+
+.suggester.suggesterDark {
```

Comment:
It can be just `.suggesterDark` because it will be harder to override it form custom stylesheet.